cachito/paths.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import shutil
+
+from pathlib import Path
```

Comment:
```suggestion
import logging
from pathlib import Path
import shutil
```

Reply:
@mprahl Is this style just for keeping used libraries in the alphabetical order?

Reply:
It's to have the following style:
```
# standard library
import os
import path

# third-party libraries
import requests

# local
import cachito.paths
```